prog/weaver/main.go

```diff
@@ -431,6 +431,12 @@ func main() {
 	if enablePlugin || enablePluginV2 {
 		go plugin.Start(httpAddr, dockerCli, pluginSocket, pluginMeshSocket, !noDNS, enablePluginV2)
 	}
+	if enablePlugin {
+		Log.Println("Creating default 'weave' network")
+		options := map[string]interface{}{plugin.MulticastOption: "true"}
+		// TODO: the driver name should be extracted from pluginMeshSocket
```

Comment:
Either.  It was hard-coded in the `weave` script before; it's hard-coded in the `.go` file now.